### DISCUSSION TITLE
How can I TOTALLY disable static thumbnail generation?

### DISCUSSION THREAD

**Discussion by Luqijian** [photoprism/photoprism#3847] [General]
Even if that I have enabled Experimental Features in settings->advanced and unchecked Preview Images in settings->library and cleaned all database or cache files, I still see thumbnail generation process when I start an complete scan Indexing task.

So how can I TOTALLY disable static thumbnail generation? I have a PhotoPrism instance running with Xeon E5-2695l V2, 64GB ram and a NVIDIA Tesla P4 GPU for acceleration. That's definitely powerful enough for Dynamic Previews feature to run seamlessly. And I have several TBs of photos and videos to be host. I don't want to wait for the endless preview generation nor tons of thumbnails to fill my NAS.

Thanks guys!

**Comment by graciousgrey** [comment]
The "Preview images" option refers to creating JPG preview images for other file types so they can be displayed in search results and in the full-screen viewer. When disabled PhotoPrism cannot index file types other than JPEG or PNG unless a preview sidecar file with the same filename prefix already exists.

In addition, PhotoPrism needs to create thumbnails, which it needs for tasks such as face recognition and image classification.

It is not possible to skip thumbnail creation entirely. 

https://docs.photoprism.app/getting-started/faq/#can-i-skip-creating-thumbnails-completely

However, you can configure PhotoPrism to dynamically render the larger thumbnails. 

https://docs.photoprism.app/user-guide/settings/advanced/#dynamic-and-static-size-limits
